config/prow/Makefile.gcloud.mk

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Google LLC
```

Comment:
With the exception of `save-kubeconfig` this seems to be doing what https://github.com/knative/test-infra/blob/main/config/Makefile does...

Reply:
Yeah we can probably just get rid of this file. I don't have a strong opinion about the `save-kubeconfig` target.